Formula/gmic.rb

```diff
@@ -20,19 +20,26 @@ class Gmic < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "4e61637e97caf2fdd854f10c685ef304731a607a8a4e0161d49b0dcfeeac08ef"
   end
 
-  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
   depends_on "fftw"
-  depends_on "jpeg"
+  depends_on "jpeg-turbo"
   depends_on "libpng"
   depends_on "libtiff"
 
+  uses_from_macos "curl"
+  uses_from_macos "zlib"
+
   def install
-    system "cmake", *std_cmake_args,
-                    "-DENABLE_FFMPEG=OFF",
-                    "-DENABLE_OPENCV=OFF",
-                    "-DENABLE_OPENEXR=OFF",
-                    "-DENABLE_X=OFF"
-    system "make", "install"
+    ENV.deparallelize
+    args = %W[
+      USR=#{prefix}
```

Comment:
Needs a fake directory to avoid trying to create GIMP plugin directory. We don't build `make gimp`, but `make install` blindly creates all directories.
```suggestion
      PLUGINDIR=#{buildpath}/plug-ins
      USR=#{prefix}
```